bot/resources/easter/save_the_planet.json

```diff
@@ -0,0 +1,71 @@
+[
+    {
+        "title": "Save the Planet: Choose Reneweable Energy",
```

Comment:
I don't think we should to include "Save the Planet: " in the embed titles, since most people already expect that. At the end of the day, they're using a `.savetheplanet` command. This applies to all titles.

Reply:
I'll change that.